.github/workflows/build-copy-and-publish-accname.yml

```diff
@@ -0,0 +1,64 @@
+name: Build, Copy, and Publish accname
+on:
+  push:
+    branches: [main]
+    paths: ['accname/**']
+  workflow_dispatch:
+
+jobs:
+  main:
+    name: Build Respec
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: w3c/spec-prod@v2
+        with:
+          SOURCE: accname/index.html
+          TOOLCHAIN: respec
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN_ACCNAME }}
+          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-aria-admin/2018Sep/0011.html
+          GH_PAGES_BRANCH: gh-pages
+          VALIDATE_WEBIDL: false
+          VALIDATE_MARKUP: false
+          BUILD_FAIL_ON: nothing
+          W3C_BUILD_OVERRIDE: |
+            specStatus: WD
+            shortName: accname-1.2
+          ARTIFACT_NAME: 'accname'
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: accname
+          path: accname.zip
+
+  copy-to-accname-gh-pages:
+    name: Copy to accname repo
+    runs-on: ubuntu-20.04
+    needs: main
+    steps:
+      - name: Checkout accname repo
+        uses: actions/checkout@v4
+        with:
+          repository: w3c/accname
+          ref: gh-pages
+          path: accname
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: accname
+          path: accname
```

Comment:
do we need `common` directory here as well?

Reply:
What I am aiming for here is trust the spec-prod action to pick up all it needs from common. To see detailed steps on how this action runs, you can see https://github.com/w3c/accname/actions/runs/9424164070/job/25963922169#step:3:531 especially where it copies the relevant assets. It then details which assets it's copying and then it uploads them as an artifact together with the generated HTML snapshot.